oar/cli/cmd_image_consistency_check.py

```diff
@@ -15,7 +15,10 @@
 @click.option(
     "-n", "--build_number", type=int, help="provide build number to get job status"
 )
-def image_consistency_check(ctx, build_number):
+@click.option(
+    "-c", "--is_nightly", is_flag=True, help="if has the flag, use candidate nightly build to test, else use stable build to test"
```

Comment:
@wewang58 maybe we don't need this shortcut. for option name can be refined like `--for-nightly`, shortcut is not required here, `-c` will make the user confuse. I cannot reflect it with nightly.

Reply:
@rioliu-rh  Updated it to for_nightly,  if using for-nightly, it will has the error:
    def image_consistency_check(ctx, build_number, for-nightly):
                                                   ^^^
 SyntaxError: invalid syntax

Reply:
@wewang58 does underscore in var name work for you